libexec/klab-report

```diff
@@ -444,7 +445,8 @@ const json_report = {
   [build_hash]: {
     date: new Date(),
     proofs,
-    git: project_HEAD
+    git: project_HEAD,
+    branch: getProjectBranch()
```

Comment:
It feels like we could start adding these things to a `git` key, rather than having multiple keys. `git.head = ...; git.branch = ...`, maybe `git.url` in the future.

Reply:
this is a good idea, but I wonder if we will run into issues with backwards compatibility with reports?

Reply:
Where would this break? You'd need to change in `overview.js` as well ofc. Are there other coupling points?

Anyway, this doesn't have to hold back this PR.